azurerm/resource_arm_sql_database.go

```diff
@@ -237,6 +285,27 @@ func resourceArmSqlDatabaseCreateUpdate(d *schema.ResourceData, meta interface{}
 		return err
 	}
 
+	if _, ok := d.GetOk("database_import"); ok {
+		if !strings.EqualFold(createMode, "default") {
+			return fmt.Errorf("database_import can only be used when create_mode is Default")
+		}
+		importParameters := expandAzureRmSqlDatabaseImport(d)
+		importFuture, err := client.CreateImportOperation(ctx, resourceGroup, serverName, name, "terraform", importParameters)
+		if err != nil {
+			return err
+		}
+
+		// this is set in config.go, but something sets
+		// it back to 15 minutes, which isn't long enough
+		// for most imports
+		client.Client.PollingDuration = 60 * time.Minute
```

Comment:
from what I recall the 15 minute reset is pulled from the API's retry interval recommendation * the number of retries allowed - I've been meaning to investigate this further